src/en/readallcomicscom/src/eu/kanade/tachiyomi/extension/en/readallcomicscom/ReadAllComics.kt

```diff
@@ -172,6 +164,22 @@ class ReadAllComics : ParsedHttpSource() {
         }
     }
 
+    private fun String.toCamelCase(): String {
```

Comment:
The function body implements PascalCase, not camelCase.

Reply:
Title Case 😅

Reply:
Doesn't Title Case imply that some words aren't capitalized? `of, and, for a, an, etc.

Reply:
yes, although it doesn't matter in this case since the title will be replaced by `mangaDetails` later

Reply:
Probably doesn't show up in categories, so it's just a matter of naming / noting the limitation properly. If you want to exclude minor words you can consider using https://github.com/ianstormtaylor/title-case-minors/blob/master/index.js

But that's enough bikeshedding from me